src/store/account/accountSlice.ts

```diff
@@ -6,30 +6,58 @@ import {
   getAccount,
   getPendingTxnList,
 } from '../../utils/appDataClient'
+import { getWallet } from '../../utils/walletClient'
+
+export type Notification = {
+  account_address: string
+  body: string
+  color?: string | null
+  footer?: string | null
+  hotspot_address?: string | null
+  hotspot_name?: string | null
+  icon: string
+  id: number
+  share_text?: string | null
+  style: string
+  time: number
+  title: string
+  viewed_at?: string | null
+}
 
 export type AccountState = {
   hotspots: Hotspot[]
+  notifications: Notification[]
   account?: Account
   mainDataLoading: 'idle' | 'pending' | 'succeeded' | 'failed'
   pendingTransactions: PendingTransaction[]
 }
 
 const initialState: AccountState = {
   hotspots: [],
+  notifications: [],
   mainDataLoading: 'idle',
   pendingTransactions: [],
 }
 
-type AccountData = { hotspots: Hotspot[]; account?: Account }
+type AccountData = {
+  hotspots: Hotspot[]
+  account?: Account
+  notifications: Notification[]
+}
 export const fetchData = createAsyncThunk<AccountData>(
   'account/fetchData',
   async () => {
-    try {
-      const data = await Promise.all([getHotspots(), getAccount()])
-      return { hotspots: data[0] || [], account: data[1] }
-    } catch (e) {
-      console.log(e)
-      return { hotspots: [] }
+    const data = await Promise.all(
+      [getHotspots(), getAccount(), getWallet('notifications')].map((p) =>
```

Comment:
So does this just load once on app start?

Reply:
for the moment, yes, but we will need to figure out a strategy around refreshing it.